src/ui/locales/fr.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Basculer l'attribution",
+    "AttributionControl.MapFeedback": "Commentaires sur la carte",
+    "FullscreenControl.Enter": "Passer en mode plein écran",
+    "FullscreenControl.Exit": "Quitter le mode plein écran",
+    "GeolocateControl.FindMyLocation": "Trouver ma position",
+    "GeolocateControl.LocationNotAvailable": "Lieu non disponible",
+    "LogoControl.Title": "Logo MapBox",
+    "Map.Title": "Carte",
+    "NavigationControl.ResetBearing": "Réinitialiser le relèvement vers le nord",
+    "NavigationControl.ZoomIn": "Agrandir",
+    "NavigationControl.ZoomOut": "Dézoomer",
+    "ScaleControl.Feet": "pi",
```

Comment:
I think `pi` is canadian only, 
```suggestion
    "ScaleControl.Feet": "pieds",
```